Shared/EmbeddedResources/ResourceLoader.cs

```diff
@@ -8,6 +9,11 @@ public static class ResourceLoader
     {
         public static string LoadString(string resourcePath)
         {
+            MessageBox.Show("loading string");
```

Comment:
I suspect the code here is temp code you added for testing? 
Again, this makes me suspect the feature is not thoroughly tested